modules/cmd_args.py

```diff
@@ -126,3 +126,4 @@
 parser.add_argument("--unix-filenames-sanitization", action='store_true', help="allow any symbols except '/' in filenames. May conflict with your browser and file system")
 parser.add_argument("--filenames-max-length", type=int, default=128, help='maximal length of filenames of saved images. If you override it, it can conflict with your file system')
 parser.add_argument("--no-prompt-history", action='store_true', help="disable read prompt from last generation feature; settings this argument will not create '--data_path/params.txt' file")
+parser.add_argument("--prepare-environment-only", action='store_true', help="launch.py argument: only prepare environment without launching webui run with --skip-torch-cuda-test")

```

Comment:
`--prepare-environment-only` seems to work the same as `--exit`
see https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Command-Line-Arguments-and-Settings#all-command-line-arguments

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/023454b49e6bf2a98804b91a12f84af2f653d128/modules/launch_utils.py#L444-L446